### DISCUSSION TITLE
Rename "Melons"

### DISCUSSION THREAD

**Discussion by TheGuizai** [BedrockTweaks/Files#93] [Ideas]
Minecraft melons are green on the outside, and red on the inside, with dark seeds. However, a real melon has shells of varied colors (but usually not as green), is typically orange or yellow on the inside, and has light seeds. As a big fan of both fruits, I know something is wrong. I suggest that melons are renamed to "watermelon," to accurately reflect their appearance, instead of retexturing their iconic appearance, since melons have varied looks, and a rename is much simpler.

**Comment by SquatchHunter** [comment]
I meant to reply to this unless my reply somehow got removed.

A rename is quite easy and will be considered a community pack so it wont be considered till then.
Our current focus is addons and getting them all back up and running
then community suggested packs.
